### ISSUE TITLE
ci: helm release process is independently versioned and released vs zipkin

### ISSUE THREAD

**Issue by codefromthecrypt** [openzipkin/zipkin#3623]
The helm release process isn't documented in build-bin/README, but looking at tags, it seems to require manual and separate automation from our normal release, which also publishes javadoc and docker.

We should not have separate versions and instead be like docker where we use the version of zipkin as the primary version. Version updates in helm should be seamless on release.

An alternative could be to move helm to another repo, where the independent versioning could make sense in the release history. However, I think it is better to just align the processes.

**Comment by codefromthecrypt** [comment]
here's an example of the odd versioning https://github.com/openzipkin/zipkin/releases/tag/zipkin-0.7.0 we release 0.3.0 then 0.7.0. The independent versioning is a cure worse than the disease, and makes sure things are not up to date unless someone remembers to make a strange version.

**Comment by codefromthecrypt** [comment]
even stranger, this makes tags called "zipkin-0.3.0" which are the tags to the helm chart.. on a repo named zipkin. This is very confusing as no one would know this is about helm.

I'll try to budget time to redo this, if I have time. but basically we should stop polluting our tag space with confusing things.

**Comment by jcchavezs** [comment]
Hi there, thanks for showing up on this. Helm was a complete community
effort driven by actual users and I also spent some time trying to automate
helm releases that is why you might find some missing releases because they
were mistaken. You can check also the PRs to see how all this happened.

Specifically talking about the tag "weirdness" you point out, an app and a
helm-chart are two different artifacts. Helm chart can change without the
actual app being changed and it felt overkill to do a zipkin release just
because helm changed. We also explored other alternatives like incremental
suffix but helm does not like other than a.b.c.

That said, we intentionally did 0.* on helm because it was still
experimental.

**Comment by codefromthecrypt** [comment]
thanks for the context @jcchavezs. this still should be fixed as the version race will continue to confuse things. It is probably best to move this to a different repository if true you can only use a tag that can eventually be mistaken for a real zipkin version. I'll ask some other folks.

**Comment by codefromthecrypt** [comment]
for context, we have been using same version of zipkin and docker since forever, even though there are times when docker has a change even when zipkin doesn't. That's an established precedent and while imperfect has resulted in both not missing docker releases, but also we have the ability to re-release docker (e.g. via the docker-x.x.x tag).

In a project like this, we have to favor existing release process and coherence, or move something that doesn't match to a different repo.

**Comment by codefromthecrypt** [comment]
TL;DR; I know this is a compromised position, but I've been told that experiments have been taking place and also that things don't work and have been blocked also. Moving this to a separate repo and deleting from here when done is the most sensible way out. I've added several recent contributors as maintainers to https://github.com/openzipkin/zipkin-helm

---

I made an executive decision to move the contents here to a new repo for reasons described above and that a couple contributors were blocked for half a year https://github.com/openzipkin/zipkin-helm I added those blocked as maintainers and also @optional303 in case there's consolidation interest from https://github.com/Financial-Times/zipkin-helm

This helps make the following true:
* helm experience is coordinated in one place and people can watch for changes only about helm in one place
* zipkin's repo gets coherent versions and tags again
* zipkin-helm can introduce features for [zipkin-dependencies](https://github.com/openzipkin/zipkin-dependencies), which is hosted in a different repository and produces different versions of docker images.
  * same is the case with [zipkin-gcp](https://github.com/openzipkin/zipkin-gcp) and [zipkin-aws](https://github.com/openzipkin/zipkin-aws)

As noted on some other issues, the final url for helm may want to be under zipkin.io. being under a different repo is fine, as right now, both zipkin and brave publish javadoc under zipkin.io.
